libsolidity/ast/ASTForward.h

```diff
@@ -57,6 +57,7 @@ class UserDefinedTypeName;
 class FunctionTypeName;
 class Mapping;
 class ArrayTypeName;
+class InlineAssembly;
```

Comment:
It doesn't seem to be used in this PR, but it was missing anyway.

Reply:
Who was assigned to me and to teach me and help me out